models/camp.py

```diff
@@ -1,6 +1,11 @@
+from models.refugee import Refugee
+from models.Volunteer import Volunteer
+
 class Camp:
     def __init__(self, name: str):
         self.name = name
+        self.refugees = set(Refugee) #Record refugees
+        self.volunteers = set(Volunteer) #Record volunteers
```

Comment:
Is this file added to this branch by mistake?
These doesn't look relevant to the issue
(And they doesn't look very right. they should be set from init args right?)

Reply:
I think the correct file would be:
from models.volunteer import Volunteer